src/components/network/network-explorer.js

```diff
@@ -37,9 +54,31 @@ const NetworkExplorer = (props) => {
         }));
     };
 
+    const handleToggleUseName = () => {
+        dispatch(toggleUseNameState());
+    };
+
+
     return (
         <div>
-            <TextField id="standard-basic" label="Search..." onChange={filter} fullWidth={true}/>
+            <Grid container spacing={2}>
+                <Grid item xs={7}>
```

Comment:
Layout is strange, toogle and filter text should both have the same width (all the width).
Also it would be more friendly to add a label to the toggle "Use name"